src/protocolsupportresourcesgenerator/generators/mappings/block/LegacyBlockDataMappingsGenerator.java

```diff
@@ -538,10 +537,10 @@ public void applyDefaultRemaps() {
 			this.<Chest>registerRemapEntryForAllStates(
 				Arrays.asList(Material.CHEST, Material.TRAPPED_CHEST),
 				o -> {
-					EnderChest enderChest = (EnderChest) Material.ENDER_CHEST.createBlockData();
-					enderChest.setWaterlogged(false);
-					enderChest.setFacing(o.getFacing());
-					return enderChest;
+					Chest chest = (Chest) o.getMaterial().createBlockData();
```

Comment:
No

Reply:
Can you explain me why older clients have to see enderchests as trapped chests, I've made this video on my server running 1.15.2 (on 4.29-dev) version and playing with 1.12.2 client.
![1 12 2-chest-test-protocolsupportrg](https://user-images.githubusercontent.com/7826998/77382399-9ae28780-6d80-11ea-9a78-28ea87ea20be.gif)

Reply:
Direct copy from discord info:

InfoBot: Q: Why do my chests look like ender chests?
A: In Minecraft 1.13, Mojang added the ability to place double chests next to each other. In order to avoid display issues and other bugs in older versions of Minecraft, all chests will now appear to be ender chests in versions of Minecraft older than 1.13.